resolwe_bio/processes/reads_processing/bqsr.py

```diff
@@ -0,0 +1,153 @@
+"""Base quality score recalibration"""
+import os
+import shutil
+
+from resolwe.process import Process, Cmd, SchedulingClass, DataField, \
+    FileField, StringField, ListField
+
+
+class BQSR(Process):
+    """
+    A two pass process of BaseRecalibrator and ApplyBQSR from GATK.
+
+    See https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.php
+    for more information.
+    """
+
+    slug = 'bqsr'
+    name = 'BaseQualityScoreRecalibrator'
+    process_type = 'data:alignment:bam:bqsr:'
+    version = '1.0.0'
+    category = 'Clipping'  # TODO: decide where to put it
+    shaduling_class = SchedulingClass.BATCH
+    entity = {'type': 'sample'}
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/dnaseq:4.2.0'
+            }
+        },
+    }
+    data_name = '{{ parameter_file.file|default("?") }}'
+
+    class Input:
+        """Input fields to perform Base quality score recalibration."""
+        bam = DataField('alignment:bam', label='BAM file containing reads', required=True)
+        reference = DataField('genome:fasta', label='Reference genome file', required=True)
+        known_sites = ListField(
+            DataField('variants:vcf', description='Known sites in (.vcf).'),
+            label='One or more databases of known polymorphic sites used to exclude regions around known '
+                  'polymorphisms from analysis.',
+            required=True
+        )
+        intervals = DataField(
+            data_type='bed',
+            label='One or more genomic intervals over which to operate. This field is opotional, but it can speed up '
+                  'the process by restricting calculations to specific genome regions.',
+            required=True
+        )
+        read_group = StringField(
+            label='Replace read groups in a BAM file.This argument enables the user to replace all read groups in the '
+                  'INPUT file with a single new read group and assign all reads to this read group in the OUTPUT BAM '
+                  'file. Addition or replacement is performed using Picard\'s AddOrReplaceReadGroups tool. Input '
+                  'should take the form of -<name><space><value> delimited by a tab, e.g. "-ID 1\t-PL Illumina\t-SM '
```

Comment:
It will be very hard for users to type a <kbd>tab</kbd> into an input

Reply:
It's not <kbd>TAB</kbd> per se, but rather its representation as `\t`. I will reflect this in the description.